src/routes/UserList.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import axios from "axios";
+import User from "../components/User";
+class UserList extends Component {
+  state = { users: [] };
```

Comment:
`user` in initialised with [], so on line # 14 if condition will always be true. That's why we aren;t seeing else case `Loading` text.